<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuoteApp</class>
 <widget class="QMainWindow" name="QuoteApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quote app</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(143, 240, 164);
border-top-color: rgb(38, 162, 105);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="fetchProverb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>388</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fetch new proverb</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="additionalField">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 18pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Author and category will be displayed here</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="proverbField">
      <property name="styleSheet">
       <string notr="true">font: 300 italic 16pt &quot;Ubuntu&quot;;
border-bottom-color: rgb(36, 31, 49);</string>
      </property>
      <property name="text">
       <string>Proverb will be displayed here</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sayProverb">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Say the proverb</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
